client/main.js

```diff
@@ -34,10 +35,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = require( 'views/root/' + rootView );
```

Comment:
Interesting. So WebPack doesn't do anything smart like ignoring unused code and modules?

Still, I would really prefer to keep it so that `requires()` take constant parameters.